community/cypher/cypher/src/test/scala/org/neo4j/cypher/CypherCompatibilityTest.scala

```diff
@@ -20,6 +20,9 @@
 package org.neo4j.cypher
 
 import java.util
+import org.neo4j.cypher.internal.RewindableExecutionResult
+import org.neo4j.cypher.internal.compatibility.ExecutionResultWrapperFor2_3
+import org.neo4j.cypher.internal.compiler.v2_3.notification.{InternalNotification, CostForUpdatesNotification}
```

Comment:
I think in this test suite you should not use the compiler internal notifications